cquadpack/bld.bat

```diff
@@ -0,0 +1,15 @@
+if "%CMAKE_GENERATOR%" == "" (

```

Comment:
Please set `CMAKE_GENERATOR` to `NMake Makefiles`. That will detect and compile this program for the right VS version.